classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = [] # Create the has-many side (a classroom has many students).
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self # Create the belongs-to side (a student belongs to a classroom).
+  end
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_associations.md):

As you have created the has-many side (a classroom has many students), kindly add a method that serves as the `belongs-to` side (a student belongs to a classroom) in `studen.rb` also.

So that any time we add a student to a classroom the classroom for the student is set as well, and vice versa. (i.e both ways relationships) 🤗